WebApplication/Backend/DTO/TimeIntervalDTO.cs

```diff
@@ -0,0 +1,79 @@
+﻿using HealthClinicBackend.Backend.Model.Util;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WebApplication.Backend.DTO
+{
+    public class TimeIntervalDTO
+    {
+        private DateTime start;
+        private DateTime end;
+        private string id;
+        public string time => Start.ToString("HH:mm") + " - " + End.ToString("HH:mm");
+        public TimeIntervalDTO()
+        {
+        }
+        public TimeIntervalDTO(DateTime start)
+        {
+            this.Start = start;
+            this.End = start.Add(new TimeSpan(0, 20, 0));
+        }
+
+        public TimeIntervalDTO(TimeInterval timeInterval)
```

Comment:
and why don't you just use Time Interval class?

Reply:
Because we need to return only DTO objects